src/plugins/vis_type_script/public/renderer/index.tsx

```diff
@@ -22,8 +23,31 @@ const getSandboxDocument = (script: string) => {
       <head>
         <meta http-equiv="content-security-policy" content="default-src none; script-src 'nonce-${nonce}' ${d3Url}">
         <script src="${d3Url}"></script>
+        <script nonce="${nonce}" type="module">
+          //  TODO: probably can't leave this using type=module
+          import { createEndpoint, fromInsideIframe } from "https://unpkg.com/@remote-ui/rpc@1.3.0/index.mjs";
+
+          const endpoint = createEndpoint(fromInsideIframe());
+
+          const KIBANA_API = {
+            searchEs: () => {
+              console.log('Calling search inside an iframe')
+              endpoint.call.searchEs();
```

Comment:
Could eventually make this a promise-based API so the visualization author can do something like 
```js
const response = await KIBANA_API.searchEs();
```